packages/web-app-files/src/components/AppBar/ViewOptions.vue

```diff
@@ -103,6 +103,15 @@ export default {
 </script>
 
 <style lang="scss" scoped>
+#files-toggle-sidebar,
+#files-view-options-btn {
+  vertical-align: middle;
+  border: 3px solid transparent;
+  &:hover {
+    background-color: var(--oc-color-background-hover);
```

Comment:
Can't see this in action when trying it out locally, somehow 🤔 sure it's applied properly?

Reply:
yes, it works in light & dark for me